wpilibc/src/main/native/cpp/simulation/Assembly2D.cpp

```diff
@@ -0,0 +1,26 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+
+
+#include "frc/simulation/Assembly2D.h"
+
+using namespace frc;
+
+
+Assembly2D::Assembly2D() : m_device{"Assembly2D"} {}
+
+void Assembly2D::SetLigamentPose(std::string ligamentPath, float angle) {
```

Comment:
Since the underlying stuff can only handle `const char*`, just use that instead of std::string.  Or we could be a bit nicer to users and use `const wpi::Twine&` and then render to the const char* via `wpi::SmallString`.